hot-desk/src/main/java/com/exadel/sandbox/config/Config.java

```diff
@@ -32,8 +28,12 @@
     }
 
     @Bean
-    public Docket api() {
+    public Docket api(TypeResolver typeResolver) {
         return new Docket(DocumentationType.SWAGGER_2)
+                .alternateTypeRules(
+                        AlternateTypeRules.newRule(typeResolver.resolve(List.class, LocalDate.class), typeResolver.resolve(List.class, java.sql.Date.class)),
```

Comment:
Do not understand this moment, u try to cast List to LocalDate?

Reply:
Nope. Swagger didn't recognize the list of LocalDate objects so I added this. Then it's working but only in some places. other places still don't recognizing object